nixos/modules/services/blockchain/ethereum/geth.nix

```diff
@@ -190,7 +199,11 @@ in
       script = ''
         ${cfg.package}/bin/geth \
           --nousb \
-          --ipcdisable \
+          ${
+            if cfg.ipc.enable
+            then ''--ipcpath ${cfg.ipc.path}''
+            else "--ipcdisable"
+          }\
```

Comment:
Keep it as one line. Also, you'll need a space before that slash at the end.

Reply:
done